testdrive/testdrive.php

```diff
@@ -57,7 +58,7 @@ function testdrive_cron($a,$b) {
 	$r = q("select * from user where account_expires_on < UTC_TIMESTAMP() + INTERVAL 5 DAY and
 		expire_notification_sent = '0000-00-00 00:00:00' ");
 
-	if(count($r)) {
+	if (DBM::is_result($r)) {
```

Comment:
Please add `use Friendica\Database\DBM;` or this will fail.

Reply:
Ops, good find!

Reply:
And should I write SQL keyword all-uppercase?

Reply:
Yes, SQL keywords should be uppercase.